pom.xml

```diff
@@ -67,14 +67,14 @@
     <maven-checkstyle.version>3.2.0</maven-checkstyle.version>
     <checkstyle.version>10.3.3</checkstyle.version>
     <prettier-maven-plugin.version>0.21</prettier-maven-plugin.version>
-    <prettier-java.version>2.2.0</prettier-java.version>
+    <prettier-java.version>2.3.1</prettier-java.version>
     <license-maven-plugin.version>4.1</license-maven-plugin.version>
 
     <!-- Spring dependencies -->
-    <spring-boot.version>2.7.14</spring-boot.version>
+    <spring-boot.version>2.7.15</spring-boot.version>
 
     <!-- Application dependencies -->
-    <kuflow.version>2.5.0</kuflow.version>
+    <kuflow.version>2.5.1-SNAPSHOT</kuflow.version>
```

Comment:
Fix before merge

Reply:
I'm not going to merge this until the upcoming SDK changes will be merged.